test/functional/sync_plan/info_test.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/sync_plan'
+
+module HammerCLIKatello
+  describe SyncPlan::InfoCommand do
+    it 'allows organization id' do
+      api_expects(:host_collections, :index) { |par| par['organization_id'].to_i == 1 }
+        .returns(index_response([{'id' => 2}]))
+
+      api_expects(:host_collections, :show) do |par|
+        par['id'].to_i == 2
+      end
+
+      run_cmd(%w(host-collection info --name hc1 --organization-id 1))
+    end
+
+    it 'allows organization name' do
+      api_expects(:organizations, :index) { |par| par[:search] == "name = \"org1\"" }
```

Comment:
You can use `OrganizationHelpers` for this. https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/organization/organization_helpers.rb